controllers/common/metering/api/v1/extensionresourceprice_types.go

```diff
@@ -26,8 +26,15 @@ const ExtensionResourcePricePrefix = "extensionresourceprice-"
 
 // ExtensionResourcePriceSpec defines the desired state of ExtensionResourcePrice
 type ExtensionResourcePriceSpec struct {
-	ResourceName string                            `json:"resourceName,omitempty"`
-	Resources    map[v1.ResourceName]ResourcePrice `json:"resources,omitempty"`
+	ResourceName      string                            `json:"resourceName,omitempty"`
+	Resources         map[v1.ResourceName]ResourcePrice `json:"resources,omitempty"`
+	GroupVersionKinds []GroupVersionKind                `json:"groupVersionKinds"`
+}
+
+type GroupVersionKind struct {
```

Comment:
Change Custom GVK to `k8s.io/apimachinery/pkg/runtime/schema.GroupVersionKind` ?

Reply:
It not have `json` tag, can't exec make install.